src/nvim/api/extmark.c

```diff
@@ -366,12 +366,10 @@ Array nvim_buf_get_extmarks(Buffer buffer, Integer ns_id, Object start, Object e
 
 /// Creates or updates an extmark.
 ///
-/// To create a new extmark, pass id=0. The extmark id will be returned.
-/// To move an existing mark, pass its id.
```

Comment:
we lost the info about "moving" a mark. that's not really made clear by the remaining text?

Reply:
Sure I added another chunk in to specify this.